assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef, useState, createRef, useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+import LeftArrowIcon from './leftArrow.svg';
+import RightArrowIcon from './rightArrow.svg';
+import GridViewIcon from './gridView.svg';
+
+const PAGE_WIDTH = 72;
+const PAGE_HEIGHT = 128;
+
+const Wrapper = styled.div`
+	position: relative;
+	display: grid;
+	grid: "left-navigation carousel right-navigation" auto / 53px 1fr 53px;
+	height: 100%;
+	width: 100%;
+	color:  ${ ( { theme } ) => theme.colors.fg.v1 };
+`;
+
+const Area = styled.div`
+	grid-area: ${ ( { area } ) => area };
+	height: 100%;
+	width: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	flex-direction: column;
+`;
+
+const List = styled( Area )`
+	flex-direction: row;
+	align-items: flex-start;
+	justify-content: ${ ( { hasHorizontalOverflow } ) => hasHorizontalOverflow ? 'space-between' : 'center' };
```

Comment:
Why `space-between`? Aren't they packed directly next to each other?

Reply:
They are. At that point it doesn't really make a difference, but `flex-start` seems more reasonable.